.rubocop.yml

```diff
@@ -0,0 +1,188 @@
+AllCops:
+  TargetRubyVersion: 2.4
+  DisabledByDefault: true
+
+#################### Layout ###############################
+
+Layout/AccessModifierIndentation:
+  Description: Check indentation of private/protected visibility modifiers.
+  StyleGuide: '#indent-public-private-protected'
+  Enabled: true
+
+Layout/IndentationWidth:
+  Description: "Use 2 spaces for indentation."
+  StyleGuide: "#spaces-indentation"
+  Enabled: true
+
+Layout/CaseIndentation:
+  Description: "Indentation of when in a case/when/[else/]end."
```

Comment:
Maybe include in the description, something like: align when/else/end/ with case. ?
To make it easier to understand what we are suppose to do about it.

Reply:
I in 99% of cases just directly copied over the description from Rubocop's internal default rule set. Not sure if it makes sense to tweak them and do a lot of changes to them. For most cases the thing that helps explaining / understanding these rules best are examples - and they can be found in the Rubocop docs or the styleguide (also referenced for each rule).

So my idea was to keep the task of explaining things to the part the knows it best - naming the docs / styleguide - and not try to explain it in detail here.

But yeah, I still can do it if you think there's a huge benefit in having it.

Reply:
Sure, let's leave it at the documentation! 👍